addons/onboarding/views/onboarding_views.xml

```diff
@@ -86,6 +95,84 @@
         </field>
     </record>
 
+    <template id="onboarding_container">
```

Comment:
:socks: we generally separate views (actual record views) from templates (e.g. in onboarding_templates.xml) as their usage is different.